packages/@textlint/ast-node-types/src/index.ts

```diff
@@ -1,11 +1,12 @@
 // MIT © 2017 azu
 "use strict";
+
 /**
  * AST Node types list on TxtNode.
  * Constant value of types
  * @see https://github.com/textlint/textlint/blob/master/docs/txtnode.md
  */
-export const ASTNodeTypes = {
+export const ASTNodeTypes: { [key: string]: string } = {
```

Comment:
📝 We can use String enum that is introduced in TypeScript 2.4.
It will be other PR. (I need more research)

```ts
export enum ASTNodeTypes {
    Document = "Document",
    Paragraph = "Paragraph",
    BlockQuote = "BlockQuote",
    ListItem = "ListItem",
    List = "List",
    Header = "Header",
    CodeBlock = "CodeBlock",
    HtmlBlock = "HtmlBlock",
    ReferenceDef = "ReferenceDef",
    HorizontalRule = "HorizontalRule",
    Comment = "Comment",
    // inline
    Str = "Str",
    Break = "Break", // well-known Hard Break
    Emphasis = "Emphasis",
    Strong = "Strong",
    Html = "Html",
    Link = "Link",
    Image = "Image",
    Code = "Code",
    Delete = "Delete"
};
```

Reply:
I've filed it as issue.
https://github.com/textlint/textlint/issues/422